test/integration/targets/certificate_complete_chain/tasks/main.yml

```diff
@@ -1,4 +1,21 @@
----
+# test code for the changed_when parameter
```

Comment:
Is there a reason why you modify `test/integration/targets/certificate_complete_chain/tasks/main.yml` in this way? If I understand this correctly, you are adding your copyright to a file you have not been involved in at all.

Reply:
Ah, no. I think I modified the wrong file there. I intended to add the copywrite thing to some other file called `main.py`. I'll undo that.

Reply:
I meant to add it to `test/integration/targets/aws_acm/tasks/main.py`. But do I even need it at all there? I see *some* `main.py` integration tests have one, some don't. Why is that?

Now that I think about it, is it even valid to put a copyright notice in any of those files? Isn't the whole repo copyrighted by the project owner? Or is it that I *own* the copyright for the files I contribute, and *license* them under GPLv3?

Reply:
I think it is the latter: you own the copyright, and license them as GPLv3.